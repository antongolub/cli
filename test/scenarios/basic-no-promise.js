'use strict'
require('../..')(main)

function main () {
  console.log('started')
  console.log('finished')
}
