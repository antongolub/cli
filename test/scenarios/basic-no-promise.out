started
finished
Error: Application entry point (main) did not return a promise.
1
